src/oscillator.js

```diff
@@ -213,6 +213,14 @@ define(function (require) {
   // these are now the same thing
   p5.Oscillator.prototype.fade =   p5.Oscillator.prototype.amp;
 
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   * 
+   * @return  {number} Amplitude value between 0.0 and 1.0
```

Comment:
I think we need `@returns` with an "s" at the end :) https://jsdoc.app/tags-returns.html

Reply:
okay ! i get , my bad